src/styles/sass/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  font-size: $base-font-size;
+  font-weight: 400;
+  border-radius: 5px;
+
+  @include smooth-animation;
+
+  &__img-container {
+    margin-top: 16px;
+    padding: 0 3px;
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__section {
+    margin-top: 16px;
+    display: flex;
+  }
+
+  &__section--description {
+    margin-top: 0;
```

Comment:
I get your idea, but you usually overwrite `margin-top` from `section` to something else. So you don't need this `section` class at all. Just add separate class for each element and add margin it's needed